lib/cloudscrape_client/dto.rb

```diff
@@ -35,10 +35,7 @@ def response(method)
         domain: domain,
         url: endpoint,
         content_type: content_type,
-        options: {
```

Comment:
Could you link to the documentation that references this API change?

Reply:
https://app.dexi.io/#/api/sections/runs/executeBulk

Reply:
☝️ That's for the `executeBulk` but as for the auth in the headers here's a good link:  https://app.dexi.io/#/api

Reply:
How's that looking @ChuckJHardyGA ?

Reply:
This change is what I am unsure about. Given your knowledge of the new API could I ask you to try update the readme to reflect the new contract and usage that allows this removal?

Another option to avoid all of the complexity added by changing the contract would be to change the interface for execute. Something like this...

```
# single (it is like this in master)
execution_id = client.runs(run_id).execute(input: { url: 'http://google.com' }, connect: true)

# multiple
execution_id = client.runs(run_id).execute(input: { urls: ['http://google.com', url: 'http://bbc.com'] }, connect: true)
```

Requires you to adapt the `urls` value to fit the bulk import contract.

Thoughts?